### ISSUE TITLE
make tests silent (the tests log are longer than the maximum Travis size)

### ISSUE THREAD

**Issue by monperrus** [SpoonLabs/coming#115]
The tests of Coming are too verbose, and the console logs reach the Travis limit.

Consequently, it is impossible to debug the Travis failure, eg https://travis-ci.org/SpoonLabs/coming/builds/559463057

We must make the tests silent.

**Comment by monperrus** [comment]
The console outputs come from `System.out.println` in both application code `src/main/java` and test code `src/test/java`.

The first thing to do is to replace the most common `System.out.println` (those polluting the output) by proper log4j log statements.
 
@niloofartrg works on this

**Comment by martinezmatias** [comment]
Perfect

**Comment by niloofartrg** [comment]
Is this issue still in progress or the logs added were sufficient? @monperrus

**Comment by monperrus** [comment]
No, there is still some work to do. We must remove all the "analyzing", "etc".

We want that the maximum log lines stays below 10000 lines to be browsed on the Travis interface.
